src/backend/storeManagers/gog/games.ts

```diff
@@ -165,48 +167,59 @@ export function onInstallOrUpdateOutput(
   totalDownloadSize = -1
 ) {
   if (!Object.hasOwn(tmpProgress, appName)) {
-    tmpProgress[appName] = defaultTmpProgress
+    tmpProgress[appName] = defaultTmpProgress()
   }
+  const progress = tmpProgress[appName]
+
   // parse log for percent
-  const percentMatch = data.match(/Progress: (\d+\.\d+) /m)
+  if (!progress.percent) {
```

Comment:
these checks were there in the original code, we need them to make sure we update only specific fields of the progress object